MedicalDocManagment.WebUI/Controllers/AdminController.cs

```diff
@@ -281,6 +281,17 @@ public IHttpActionResult GetPosition(int id)
         }
         #endregion
 
+        #region Mkh's methods
+
+        [HttpGet]
+        public IHttpActionResult GetClassesMkh()
```

Comment:
I've done it when I was resolving merge conflict with dev.
Next commit is going to contain it.